api/pom.xml

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
```

Comment:
can you fix the header to be the standard license header instead of the old-style header?